ensure_tiller_installed.go

```diff
@@ -158,6 +158,7 @@ func (c *Client) EnsureTillerInstalledWithValues(ctx context.Context, values []s
 
 	i := &installer.Options{
 		AutoMountServiceAccountToken: true,
+		ForceUpgrade:                 true,
```

Comment:
Since we need to type `--force-upgrade` 
```
helm --tiller-namespace giantswarm --kube-context giantswarm-v86hg init --upgrade --force-upgrade --tiller-image gcr.io/kubernetes-helm/tiller:v2.12.0
$HELM_HOME has been configured at /Users/jihyukbok/.helm.
```